client/components/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "@automattic/simple-components",
+	"version": "0.1.0-alpha.6",
+	"description": "React components, as used on WordPress.com",
+	"scripts": {
+		"build": "npx babel --only ./index.js,./badge/,./button/,./main/,./notice/ --out-dir cjs .",
```

Comment:
It's super simple when you compare to what we offer in Gutenberg:
https://github.com/WordPress/gutenberg/blob/master/bin/packages/build.js

We might want to sync on this and expose one general script.

Reply:
I like simple :grin: TBH, those build scripts looked a bit... overblown? :grin: To me, it seemed like the most relevant config options were (declarative) `.babelrc` material (rather than -- imperative -- script material). I think I'd like us at least to wait for the Babel 7-beta.53 upgrade (#26020) to see how far we can get using Babel configs.

Reply:
Our build scripts are inspired by Jest and they are built the way they are mainly because we want quick "watch" support, which means if there's a change in a file we only transpile this file. Maybe it's possible to simplify this as well but yeah let's try to align once the project matures.